### PR TITLE
Fix for grouping & min column width

### PR DESCRIPTION
**Pull request by MatijaA**
The grid didn't work properly when no grouping column or minWidth was selected.
Removed autobind.

### CODE REVIEW

File: src/components/ServerGridDashboard/ServerGridDashboard.tsx [hunk 1]
```diff
@@ -118,9 +118,7 @@ export class ServerGridDashboard extends React.Component<IServerGridDashboardPro
         this.state = {
             rows: this.transformFarmToRows(props.farms, props.filter),
             expandedRows: {},
-            groupBy: ['FarmName'],
-            sortColumn: 'ServerName',
```
Comment (hbagarich) [inline src/components/ServerGridDashboard/ServerGridDashboard.tsx]: Why did you remove sort configuration?

Comment (MatijaA) [inline src/components/ServerGridDashboard/ServerGridDashboard.tsx]: The sort configuration is in the Grids state and has no use in ServerGridDashboard
